geoportailv3/templates/index.html

```diff
@@ -54,7 +54,8 @@
             </ul>
           </li>
         </ul>
-        <app-search app-search-map="::mainCtrl.map"></app-search>
+        <app-search app-search-map="::mainCtrl.map"
+                    app-search-mobile-active="mainCtrl.mobileSearchActive"></app-search>
```

Comment:
We never use four-word attribute names elsewhere. So I'd use `app-search-mobileactive`.